docs: update multiple aliases for sub-accounts (#172)
lacework/terraform-provider-lacework: website/docs/index.html.markdown

}
```

From there, you can pass the [`alias` meta-argument](https://www.terraform.io/docs/language/providers/configuration.html#alias-multiple-provider-configurations) to any resource or module to switch between accounts.

For resource blocks you can can reference the `alias` using : 
```hcl
resource "lacework_alert_channel_slack" "primary_critical" {
  provider = lacework.primary
  # ...
}

resource "lacework_alert_channel_slack" "business_unit_critical" {
  provider = lacework.business-unit
  # ...

> }business-unit

Reviewer: typo? 🤔

Author: I dont see it

```

For module blocks you can can reference the `alias` using :
```hcl
module "lacework_module" {
  source  = "lacework/s3-data-export/aws"
  version = "~> 0.1"
  providers = {
  lacework = lacework.primary
  }
  # ...
}

module "lacework_module" {
  source  = "lacework/s3-data-export/aws"
  version = "~> 0.1"
  providers = {
  lacework = lacework.business-unit
  }
 # ...

> }

Reviewer: I am not sure we should add documentation about modules in the provider, let me think about this for a min.

Author: Ok makes sense happy to take it to the module

Author: Let me know your thoughts here, happy to update the module documentation as needed for everything

Reviewer: leave them.

}
```

From there, you can pass the [`alias` meta-argument](https://www.terraform.io/docs/language/providers/configuration.html#alias-multiple-provider-configurations) to any resource or module to switch between accounts.

> For resource blocks you can can reference the `alias` using : 

Reviewer: I don't think "blocks" is the right word here. 
```suggestion
For a resource or data source, set its `provider` meta-argument to a `lacework.<ALIAS>` reference:
```

```hcl
resource "lacework_alert_channel_slack" "primary_critical" {
  provider = lacework.primary
  # ...
}

resource "lacework_alert_channel_slack" "business_unit_critical" {
  provider = lacework.business-unit
  # ...
}
```

> For module blocks you can can reference the `alias` using :

Reviewer: ```suggestion
For a module, use its `providers` meta-argument to specify which provider configurations should be mapped to which local provider names inside the module:
```

```hcl
module "lacework_module" {
  source  = "lacework/s3-data-export/aws"
  version = "~> 0.1"

>   providers = {
>   lacework = lacework.primary
>   }

Reviewer: ```suggestion

  providers = {
    lacework = lacework.primary
  }
```

  # ...
}

module "lacework_module" {
  source  = "lacework/s3-data-export/aws"
  version = "~> 0.1"

>   providers = {
>   lacework = lacework.business-unit
>   }

Reviewer: same, fix indentation
```suggestion

  providers = {
    lacework = lacework.business-unit
  }
```
